Description: Make build reproducible by using SOURCE_DATE_EPOCH
Author: Nilesh Patra <npatra974@gmail.com>
Forwarded: no
Last-Update: 2020-12-24
--- a/rollup.config.js
+++ b/rollup.config.js
@@ -3,7 +3,7 @@
 import {terser} from "rollup-plugin-terser";
 import * as meta from "./package.json";
 
-const copyright = `// ${meta.homepage} v${meta.version} Copyright ${(new Date).getFullYear()} ${meta.author.name}`;
+const copyright = `// ${meta.homepage} v${meta.version} Copyright ${(new Date(process.env.SOURCE_DATE_EPOCH ? (process.env.SOURCE_DATE_EPOCH * 1000) : new Date().getTime())).getFullYear()} ${meta.author.name}`;
 
 function onwarn(message, warn) {
   if (message.code === "CIRCULAR_DEPENDENCY") return;
